### PR TITLE
[SRVCOM-2121] Add the marketplace prerequisite for the Serverless operator

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/SRVCOM-2121

Link to docs preview:
https://51572--docspreview.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html#serverless-install-web-console_install-serverless-operator

(the second requirement entry)

https://51572--docspreview.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html#additional-resources_install-serverless-operator

(the second additional resource entry)

QE-approved (in https://github.com/openshift/openshift-docs/pull/51075#pullrequestreview-1138967587 )

### CODE REVIEW

**Review by cardil — APPROVED** [review]
/lgtm

from QE

File: modules/serverless-install-web-console.adoc [hunk 1]
```diff
@@ -12,6 +12,7 @@ You can install the {ServerlessOperatorName} from the OperatorHub by using the {
 
 ifdef::openshift-enterprise[]
 * You have access to an {product-title} account with cluster administrator access.
+* Your cluster has marketplace capability enabled or Red Hat catalog source configured manually.
```
Comment (abhatt-rh) [inline modules/serverless-install-web-console.adoc:15]: Consider the following term usage for [marketplace](https://docs.openshift.com/container-platform/4.11/applications/red-hat-marketplace.html) and [catalog source](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm/olm-understanding-olm.html#olm-catalogsource_olm-understanding-olm). 
```suggestion
* Your cluster has Red Hat Marketplace capability enabled or Red Hat-provided catalog source configured manually.
```

**Review by abhatt-rh — COMMENTED** [review]
Hi @rh-max, I left a comment regarding the usage of some terms!

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/serverless-install-web-console.adoc (hunk 1 above)
Comment (cardil) [reply modules/serverless-install-web-console.adoc:15]: The [Red Hat Marketplace](https://docs.openshift.com/container-platform/4.11/applications/red-hat-marketplace.html) is a totally different, and unrelated thing. We are taking, here, about OCP's ["marketplace capability"](https://docs.openshift.com/container-platform/4.11/post_installation_configuration/cluster-capabilities.html).

+1 for Red Hat-provided catalog source

Comment (Preeticp) [reply modules/serverless-install-web-console.adoc:15]: @cardil @rh-max ideally we should keep the wording consistent across our Operators. Please see the PR for Pipelines, SBO and GitOps: https://github.com/openshift/openshift-docs/pull/51736/files#diff-83498e11a11b3a969b166ee1dc9a91409ee6cd52c6ce3f3a1bd85a1834349ed3R20
Also, we need to link the prerequisite to https://docs.openshift.com/container-platform/4.11/post_installation_configuration/cluster-capabilities.html 
(Note: The other PR needs to change the link to an xref)

Comment (rh-max) [reply modules/serverless-install-web-console.adoc]: @Preeticp Didn't initially see your comment -- I implemented it now.

**Pull request merged**